assets/gpt-4-turbo_config.yml

```diff
@@ -0,0 +1,43 @@
+api_keys:
+  gpt-3.5-turbo-1106:
+    - api_key: sk-xxxxx
+      organization: org-xxxxxx
+      model: gpt-3.5-turbo-1106
+  gpt-4-1106-preview:
+    - api_key: sk-xxxxx
+      organization: org-xxxxxx
+      model: gpt-4-1106-preview
+
+
+default_request_type: openai # or xagent
+default_completion_kwargs:
+  model: gpt-4-1106-preview
+  temperature: 0.2
+  request_timeout: 60
+
+enable_summary: true
+summary:
+  single_action_max_length: 2048
+  max_return_length: 126464 # https://github.com/OpenBMB/XAgent/pull/297#discussion_r1413349570
```

Comment:
`summary.max_return_length` + `max_plan_length` < 128k - `extra_cost(1000)` - 4096 = 7704 tokens.

But right now `126464(max_return_length)` + `4096(max_plan_length)` > 7704?

Reply:
You're right, we should use 122904.

gpt-4-32k has 32768 tokens (32 * 1024)
gpt-4 is 8k with 8192 tokens (8 * 1024)
gpt-3.6-turbo-16k is 16385 is (16 * 1024 + 1)
but gpt-4-1106-preview has [4096 return tokens and 128000 total](https://platform.openai.com/docs/models/model-endpoint-compatibility) not 128 * 1024 = 131072.